Framework/CurveFitting/src/FuncMinimizers/DampedGaussNewtonMinimizer.cpp

```diff
@@ -30,6 +30,7 @@ DECLARE_FUNCMINIMIZER(DampedGaussNewtonMinimizer, Damped GaussNewton)
 DampedGaussNewtonMinimizer::DampedGaussNewtonMinimizer(double relTol)
     : IFuncMinimizer(), m_relTol(relTol) {
   declareProperty("Damping", 0.0, "The damping parameter.");
+  declareProperty("Debug", false, "Turn on the debug output.");
```

Comment:
Does the explicit debugging property really make sense? We do have a `debug` channel in the logger already.